<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ns::SettingsHintPage</class>
 <widget class="QWidget" name="ns::SettingsHintPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>397</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Hint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../translator.qrc">
    <normaloff>:/images/ui/act_hint.png</normaloff>:/images/ui/act_hint.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: Verdana, &quot;宋体&quot;, Arial, Sans-serif, Segoe UI, Tahoma;
}

#widget {
	background: #F2F9FF;
}

#framePromptLine {
	background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,stop: 0 #80ACCF, stop: 1.0 #F2F9FF);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxCloseHint">
        <property name="text">
         <string>Close all voice prompt</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="labelHintDescription">
          <property name="text">
           <string>Prompt Setting</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="framePromptLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="labelPromptTitle">
          <property name="text">
           <string>You can modify the prompt setting</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>13</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxCallPrompt">
            <property name="text">
             <string>Voice prompt when call incoming</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxMessagePrompt">
            <property name="text">
             <string>Voice prompt when receive message</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>286</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../translator.qrc"/>
 </resources>
 <connections/>
</ui>
